### DISCUSSION TITLE
Hidden collections

### DISCUSSION THREAD

**Discussion by Sendiexperiment** [phantom/docs#140] [General]
Hey there, what determines if certain NFTs are placed in hidden collections? Also, what are the criteria for flagging them?

Please propose some solution, my team is launching a collection for a web2 brand

4KmACR7weNy5sqihMrpqJ2naQwQDfKqLbWCjjygf4zpu update authority

**Comment by adamdelphantom** [comment]
Hey @Sendiexperiment,

Thanks for writing in! I'm looking into this and will follow up once I have progress.

**Reply by adamdelphantom** [reply]
This has since been resolved.
